testsuite/MDAnalysisTests/lib/test_distances.py

```diff
@@ -57,7 +57,7 @@ def test_capped_distance_noresults():
 npoints_1 = (1, 100)
 
 boxes_1 = (np.array([1, 2, 3, 90, 90, 90], dtype=np.float32),  # ortho
-           np.array([1, 2, 3, 30, 45, 60], dtype=np.float32),  # tri_box
+           np.array([1, 2, 3, 45, 60, 90], dtype=np.float32),  # tri_box
```

Comment:
do the tests fail with the 30 angle?

Reply:
No, It was some misunderstanding. I will change it again.